pacific-1/proposals/08-11-2023-mint-schedule.json

```diff
@@ -0,0 +1,58 @@
+{
+  "title": "Mint Param Change Proposal",
+  "description": "Proposal to add Daily Mint Token Release Schedule",
+  "changes": [
+    {
+      "subspace": "mint",
+      "key": "TokenReleaseSchedule",
+      "value": [
+	    {
+	      "token_release_amount": "300000000000000",
+	      "start_date": "2023-08-15",
+	      "end_date": "2024-08-14"
+	    },
+	    {
+	      "token_release_amount": "225000000000000",
+	      "start_date": "2024-08-15",
+	      "end_date": "2025-08-14"
+	    },
+	    {
+	      "token_release_amount": "180000000000000",
+	      "start_date": "2025-08-15",
+	      "end_date": "2026-08-14"
+	    },
+	    {
+	      "token_release_amount": "165000000000000",
+	      "start_date": "2026-08-15",
+	      "end_date": "2027-08-14"
+	    },
+	    {
+	      "token_release_amount": "135000000000000",
+	      "start_date": "2027-08-15",
+	      "end_date": "2028-08-14"
+	    },
+	    {
+	      "token_release_amount": "120000000000000",
+	      "start_date": "2028-08-15",
+	      "end_date": "2029-08-14"
+	    },
+	    {
+	      "token_release_amount": "112500000000000",
```

Comment:
missing one more entry of 112500000000000

Reply:
i believe the last item should be ending in 2033 right? so the last two need to be shifted and this one needs to be inserted